docs/source/admin.rst

```diff
@@ -97,6 +97,14 @@ created
   ``management``, ``testcases``, ``testplans`` and ``testruns`` applications.
   These are the permissions required for normal usage of Kiwi TCMS!
 
+To ensure that all necessary permissions are assigned to the default
+group (Tester) and that the stale permission objects are removed, after
+the system installation or upgrade use the following management
+command::
+
+    ./manage.py refresh_permissions
```

Comment:
TODO: this will also have to incorporate a call to `clean_orphan_obj_perms` from django-guardian once #1911 gets metged.